.travis.yml

```diff
@@ -3,7 +3,7 @@ sudo: false
 install:
 - pip install --user mkdocs==0.17.5 mkdocs-material==2.9.4
 script:
-- mvn test -B
+- mvn clean verify --batch-mode
```

Comment:
What is the reason for switching to ```verify``` phase. Do we have issues caused by stopping the build at ```test```?

The commit message does not tell the truth. According to the commit message title I would not expect the switch to verify.

Reply:
:confused: changed it to execute JaCoCo reporting..